pages/live/lobby.js

```diff
@@ -186,7 +186,7 @@ async function lobbyInit () {
 
   // Connect to the WebSocket
   if (lobbySocket) lobbySocket.close();
-  lobbySocket = new WebSocket("wss://epochtal.p2r3.com:8080/ws/lobby_" + encodedName);
+  lobbySocket = new WebSocket(`wss://${window.location.host}/ws/lobby_${encodedName}`);
```

Comment:
Suggestion: this could rely on a check for HTTPS from `window.location.protocol` to decide between `ws` and `wss`, so that we're not blindly assuming a secure protocol.